src/Shared/FileMatcher.cs

```diff
@@ -2457,6 +2457,10 @@ private static string[] CreateArrayWithSingleItemIfNotExcluded(string filespecUn
                             // The suffix is the same so there is a possibility that the two will match the same files.
                             searchesToExclude.Add(excludeState);
                         }
+                        else
+                        {
+                            Console.Write("Hoho");
```

Comment:
no Hoho :)

Reply:
And now you know how I debug my code :D Removed!